src/wirecraft/client/game.py

```diff
@@ -239,12 +258,7 @@ def update_zoom(self):
 
     def handle_left_click(self, camera: Camera) -> None:
         """Handle left mouse button click."""
-        if not self.is_placing_cable:
-            pass
-            # self.start_cable_connection(camera)
-        else:
-            pass
-            # self.end_cable_connection(camera)
+        self.click_pos = pygame.mouse.get_pos()
```

Comment:
What is the point of self.click_pos if mouse pos can be accessed from pygame.mouse.get_pos() at any point ?

edit: maybe I misunderstood and the self.click_pos is fix until button is released ?